src/content/about/marketing.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Marketing activities
+description: Accepted practices for corporate involvement in the nf-core community
+---
+
+This page is intended for anyone coming to the Nextflow and nf-core community when representing a company or corporate interest.
+
+Firstly - welcome! We are an open and inclusive community and encourage everyone to be involved. Here we try to set out some guidelines to act as guard-rails for activity, both for the benefit of community members but also so that you feel comfortable that you are interacting with the community in acceptable ways.
+
+## Code of conduct
+
+The [code of conduct](/code_of_conduct/) applies to everyone active in the community.
+Two points are particularly worth highlighting, which are not allowed:
+
+- Sending unsolicited direct-messages or emails solely for marketing purposes to community members.
+- Posting in nf-core spaces specifically to criticise or detract from Nextflow and nf-core solely for the purposes of promoting alternatives.
+
+Contravening these, or any of the behaviours listed in the code of conduct, will likely result in an immediate ban.
+
+## Slack channels
+
+Please refrain from posting about your products and services in Slack, with some exceptions:
+
+- The [`#shameless-promo`](https://nfcore.slack.com/channels/shameless-promo) channel
+  - This channel is designed specifically for promotion of personal and commercial projects relevant to the nf-core community.
+  - "Relevant" means that a significant chunk of the people reading the message may find it useful for their daily work.
+  - See above code of conduct points. Posting that tool XYZ is better than Nextflow / nf-core and advertising it as an alternative is not ok.
+- In reply to active discussions
+  - If your tool or product is relevant to an ongoing discussion, it's ok to mention it.
+  - For example: _"The things you're trying to achieve can be done with tool XYZ, let me know if you would like more details."_
+  - Please use common sense: don't swamp the thread and push too hard.
+
+Posts that the nf-core core team / safety team deem to violate these principles will be deleted, and the authors may be banned.
+
+If in doubt, please reach out to one of the [core team](/governance#core-team) to ask if your idea is ok or not.
+
+## Slack direct messages
+
+Unsolicited direct messages (DMs) are explicitly forbidden in our community.
+
+For example, sending DMs to community members such as the following is _not_ ok:
+
+> Hi XXX! I work at YYY and we're building the future of computational biology.
+> We're launching a new tool and are looking for beta testers, and you seem
+> to match the profile that we're looking for. Please let us know if you'd
+> like us to sign you up!
+
+Please use public channels such as [`#shameless-promo`](https://nfcore.slack.com/channels/shameless-promo) instead of doing this.
```

Comment:
Also, update here with "marketing purposes". Actually, I will state that sending DMs it is forbidden for marketing purposes, and for others it is strongly suggested.

Reply:
Done in https://github.com/nf-core/website/pull/1992/commits/8b997cb2c57a9316d35827bd910c60722ddb9140